File: WebApplication/Models/Group.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication.Models
{
    public class Group : EntityBase<Group>
    {
        public string Name { get; set; }
        //public List<Student> Students { get; set; }
        //public List<GroupCourse> GroupCourses { get; set; }
    }
}

File: SparkApplication/src/main/java/Models/Group.java
package Models;

public class Group extends EntityBase {

    private String name;

    public Group(String id,String type,String name){
        super(id,type);
        this.name = name;
    }

    public Group(){
        super();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return getId() + "," + getType() + "," + name;
    }


}
